assets/scripts/datalayer/common.js

```diff
@@ -113,7 +113,16 @@ document.shopAnalytics = {
       if (shop_analytics_settings.datalayer_console_log === 'on') {
         console.dir(event_data);
       }
-      window.dataLayer.push(event_data);
+      if (shop_analytics_settings.tc_enabled == 1) {
+        window.tc_vars = window.tc_vars || [];
```

Comment:
```suggestion
        window.tag_commander_vars = window.tag_commander_vars || [];
```

Reply:
tc_vars is actually predefined by the tag commander, thats why i used it.